layout/borderlayout.go

```diff
@@ -1,7 +1,9 @@
 package layout
 
-import "github.com/fyne-io/fyne"
```

Comment:
I agree that this formatting was incorrect - but it would be better to separate format changes from code changes - these files did not change in code at all so it can confuse the history...

Reply:
I have updated all of the project imports as a single change so this should not be a problem again